.github/workflows/pkgci_regression_test_amdgpu_rocm.yml

```diff
@@ -75,3 +75,77 @@ jobs:
               -rpfE --timeout=30 --retries=2 --durations=20 \
               --config-files=build_tools/pkgci/external_test_suite/onnx_gpu_rocm_rdna3.json \
               --no-skip-tests-missing-files
+  linux_x86_64_rocm_models:
+    name: Linux (x86_64) AMDGPU Rocm Model Testing + Benchmark
```

Comment:
Put MI250 in these names.

Consider these names 
`PkgCI / Regression Test AMDGPU-Vulkan / Linux (x86_64) Model Testing`
`PkgCI / Regression Test AMDGPU-ROCm / Linux (x86_64) AMDGPU Rocm Model Testing + Benchmar`

...as they appear in checks:
![image](https://github.com/iree-org/iree/assets/4010439/1e9cc41e-e6ce-4ac3-8936-c37636b490df)

... and on the 'run logs' page:
![image](https://github.com/iree-org/iree/assets/4010439/77a5bb52-d4d9-4a42-a556-0a5694ccf77e)

The names are long and have some redundant information.

How about `Linux MI250 - Models` here, for `PkgCI / Regression Test AMDGPU-ROCm / Linux MI250 - Models`. Or just `MI250 - Models`